var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -80,11 +88,19 @@ class Cp2k(MakefilePackage, CudaPackage):
     depends_on('libxsmm@1.11:~header-only', when='smm=libxsmm')
     # use pkg-config (support added in libxsmm-1.10) to link to libxsmm
     depends_on('pkgconfig', type='build', when='smm=libxsmm')
+    # ... and in CP2K 7.0+ for linking to libint2
+    depends_on('pkgconfig', type='build', when='@7.0:')
 
     # libint & libxc are always statically linked
-    depends_on('libint@1.1.4:1.2', when='@3.0:', type='build')
+    depends_on('libint@1.1.4:1.2', when='@3.0:6.9', type='build')
```

Comment:
We should probably add a conflict with a sensible message if people use a value for `lmax` which is not the default and a version of `cp2k` less than 7.

Reply:
good point, I think it makes sense to add a general `conflicts('tune!=none', when='@:2.5.99')`, since we ignore it for libint < 2.6, which then also solves the issue with cp2k.

Reply:
I had to factor out the list of tune-options variable, but now there is a conflict if the user tries to install libint prior to 2.6 with a variant other than `none`

Reply:
I think that's also a good solution as long as #9740 is not implemented. My idea was also to forbid people from changing the default `lmax` for old versions of `cp2k` - but that is also suboptimal as in that case what we really want is for the variant NOT to be there.